code/game/turfs/space.dm

```diff
@@ -84,4 +84,8 @@
 	. = ..()
 	if(isliving(AM))
 		var/mob/living/spaceman = AM
+
+		if(spaceman.status_flags & GODMODE) //Invulnerable mobs shouldn't care/take damage
+			return
+
```

Comment:
Yes they should space is killtiles for a reason

Reply:
No they shouldn't; if you're in godmode, why should you take damage? If you don't want them to go into space tiles period, just say that.

Reply:
if you're in a space tile you should die period